test/show.jl

```diff
@@ -569,7 +569,7 @@ let repr = sprint(dump, Any)
     @test length(repr) > 100000
     @test ismatch(r"^Any\n  [^ \t\n]", repr)
     @test endswith(repr, '\n')
-    @test_broken contains(repr, "     Base.Vector{T} = Array{T,1}\n")
+    @test contains(repr, "     Base.Vector{T} = Array{T,1} where T\n")
```

Comment:
It would be more accurate print either `Base.Vector = Array{T,1} where T` or `Base.Vector{T} = Array{T,1}`. That second syntax will be added soon.